src/authenticated/project/ProjectEditPage.js

```diff
@@ -48,7 +50,8 @@ const ProjectEditPage = compose(
         mutate({
```

Comment:
We want this `updateProjectQuery` graphql HOC to refetch `getUser` since those queries could now have stale data after this mutation happens i.e.

```js
graphql(updateProjectQuery, {
// ..
  options: {
    refetchQueries: ['getUser']
  }
})

Reply:
Ok, does getUser need to be defined in the updateProjectQuery as well or is that handled somewhere else already?

Reply:
Nope, it's the name of a graphql function (not sure what the actual terminology is). You can see an example of a query using it [here](https://github.com/codefordenver/members/blob/a9c1e136ccfca82257e7f452512eb95d46948635/src/authenticated/member/MemberProfilePage.js#L6). It can just be referenced by string and apollo will mark all the queries that request it as stale and refetch them if that data is used in a rendered component